src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,165 @@
 Release History
 ===============
 
+2.49.0
+++++++
+
+**ACR**
+
+* `az acr create`: Remove `Classic` from `sku` option (#26116)
+
+**AKS**
+
+* [BREAKING CHANGE] `az aks create`: Specifying `--pod-cidr` with Azure CNI will return an error instead of logging a warning when not use `overlay` mode (#26237)
```

Comment:
```suggestion
* [BREAKING CHANGE] `az aks create`: Specify `--pod-cidr` with Azure CNI will return an error instead of logging a warning when not use `overlay` mode (#26237)
```